src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        final int index = random.nextInt(5);
```

Comment:
this and all below magic values should be constants with proper naming

Reply:
![image](https://user-images.githubusercontent.com/79547153/227925358-6d42d34e-cf75-4efd-bd8f-e5f41eb5702c.png)
![image](https://user-images.githubusercontent.com/79547153/227925426-60cec37d-585c-4c5a-ab1e-75cfe77597cf.png)
i made it constants and got this